lntest/harness.go

```diff
@@ -648,6 +649,57 @@ func (n *NetworkHarness) StopNode(node *HarnessNode) error {
 	return node.stop()
 }
 
+// SaveProfilesPages hits profiles pages of all active nodes and writes it to
+// disk using a similar naming scheme as to the regular set of logs.
+func (n *NetworkHarness) SaveProfilesPages() {
+	for _, node := range n.activeNodes {
+		resp, err := http.Get(
+			fmt.Sprintf(
+				"http://localhost:%d/debug/pprof/goroutine?debug=1",
+				node.cfg.ProfilePort,
+			),
+		)
+		if err != nil {
+			fmt.Printf("Failed to get profile page: "+
+				"NodeId is %d; Node name is %s. "+
```

Comment:
Nit: would make this `id=%d, name=%s, pubkey=%x`

Reply:
could also apply to the other error messages in this method

Reply:
Done